### ISSUE TITLE
Training the model on QM9.

### ISSUE THREAD

**Issue by renatofrazzato** [atomistic-machine-learning/schnetpack#620]
When I try to train the model on QM9 I got an error: ValueError: dictionary update sequence element #0 has length 1; 2 is required. When I try to run the line: trainer.fit(task, datamodule=qm9data)

I am using the exactly the same code from the tutorial notebook which was working some months ago.

Thanks in advance.

**Comment by jnsLs** [comment]
Dear @renatofrazzato,
are you talking about that tutorial: `tutorial_02_qm9` ?
For me it runs without problems. You could try to delete the qm9.db file and the split.npz file from the previous runs and run the tutorials again. If that does not solve your issue, I would need more details, to help you solve it.
Best,
Jonas

**Comment by renatofrazzato** [comment]
Dear @jnsLs., thank you for your support.
Yes, I was talking about tutorial_02_qm9 and I am using google colab to run the code. I could run the code again, I changed the way to install the SchnetPack I was installing the library using the command "!pip install git+https://github.com/atomistic-machine-learning/schnetpack.git" and now I tried simply "!pip install schnetpack" and it worked.

Thanks, Renato
